cryptoinvestor/models/user.py

```diff
@@ -7,3 +8,44 @@ def __init__(self, *, username: str, password: str, account: Account):
         self.username = username
         self.password = password
         self.account = account
+        self.id = id
+        self.rates = {}
+        self.coin_status = 10000
```

Comment:
This I think should mean how much balance user has. But there already is a variable for that (User has account and account has balance). If you want to change it please remove the account class. It shouldn't be on multiple places anyway.